x-pack/plugins/watcher/public/application/sections/watch_edit_page/watch_edit_page.tsx

```diff
@@ -154,14 +150,14 @@ export const WatchEditPage = ({
 
   if (!watch) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+      <EuiPageTemplate.EmptyPrompt>
```

Comment:
Added `EuiPageTemplate.EmptyPrompt` otherwise the loading prompt doesn't have the color.
<img width="300" alt="Screenshot 2023-08-22 at 13 33 51" src="https://github.com/elastic/kibana/assets/6585477/f152b224-32a6-4be8-9352-e8e345d65204">